offlineimap/imapserver.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, repos):
             self.af = socket.AF_INET
         else:
             self.af = socket.AF_UNSPEC
-        self.hostname = None if self.preauth_tunnel else repos.gethost()
+        self.hostname = None if self.transport_tunnel else repos.gethost()
```

Comment:
Does this mean we should require remotehost when preauth_tunnel is used?

Reply:
I would say yes, because the `preauth_tunnel` is only used for authentication and not for an imap connection itself? But the `transport_tunnel` provides open an imap interface which is used instead of the hostname.

Reply:
Ok. Please, send your signed-off so I can apply.